cli/kata-check.go

```diff
@@ -307,14 +310,71 @@ var kataCheckCLICommand = cli.Command{
 	Usage: "tests if system can run " + project,
 	Flags: []cli.Flag{
 		cli.BoolFlag{
-			Name:  "verbose, v",
-			Usage: "display the list of checks performed",
+			Name:  "check-version-only",
+			Usage: "Only compare the current and latest available versions (requires network, non-root only)",
```

Comment:
A minor nit.. It's not obvious from the description that it checks for `release` versions only. Although the subsequent options help with the clarification. Just noting down in case if it makes sense to make it more explicit.